static/app/components/forms/fields/booleanField.tsx

```diff
@@ -33,10 +35,9 @@ export default class BooleanField extends Component<BooleanFieldProps> {
     const {confirm, ...fieldProps} = this.props;
 
     return (
-      <InputField
-        {...fieldProps}
-        resetOnError
-        field={({
+      <FormField {...fieldProps} resetOnError>
+        {({
+          children: _children,
```

Comment:
The only thing InputField was really doing was avoiding `children` from being passed down. 

I think we actually stop this from happening in the `FormField` itself, and remove all of these destructuring prop removals of children. But for now, in all of these form fields I will just keep the destructuring to be safe.